libs/frontend/feature-build/src/components/toolbar/ToolbarContainer.vue

```diff
@@ -27,8 +27,8 @@
       <SizeDropdown />
       <ToolbarColorPicker
         :tooltip="t('style.toolbar.background_color')"
-        :color="getStyleValue(Css.BackgroundColor)"
-        :gradient="getStyleValue(Css.Background)"
+        :color="getRawStyle(Css.BackgroundColor)"
```

Comment:
The original style with theme variables is passed to the color picker now, so I added a function to get the style without resolving theme.